### PR TITLE
add common test

### PR DESCRIPTION
**Pull request by zmstone**
TODO, show code coverage summary in travis.

### CODE REVIEW

**Comment by id** [comment]
Why not run unit tests as unit tests + add common test suites?

**Comment by zmstone** [comment]
because then you'll need to merge coverage data.

**Comment by zmstone** [comment]
@id i think now it's enough testing for producer.
currently travis is not happy because the patch for supervisor3 is not yet merged.

please review this pr.

File: scripts/cover-print-not-covered-lines.escript [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  io:format("using coverdata file: ~s\n", [UtCoverDataFile]),
+  {ok, CtCoverDataFile} = find_latest_coverdata(CtCoverDataDir),
+  io:format("using coverdata file: ~s\n", [CtCoverDataFile]),
+  Parent = self(),
+  Ref = make_ref(),
+  erlang:spawn_link(
+    fun() ->
+      %% shutup the chatty prints from cover:xxx calls
+      {ok, F} = file:open("/dev/null", [write]),
+      group_leader(F, self()),
```
Comment (id) [inline scripts/cover-print-not-covered-lines.escript:40]: nice :)

**Comment by id** [comment]
Fantastic! Thanks.

**Pull request merged**